README.md

```diff
@@ -37,7 +37,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Click here for live video Demo]( Add the deployed Link in here )
+- [Click here for live video Demo](https://github.com/Brian3Kamau/Brian3Kamau.github.io)
```

Comment:
Please consider adding a proper link to the live version of your portfolio to meet the project requirements. Currently, clicking on the link reopens your repository rather than your portfolio. 

## Remember:-
 When making another request for review first verify by clicking on the live demo link of the portfolio which you made live with GitHub pages.